com.endlessm.Showmehow.json.in

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "com.endlessm.Showmehow",
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "command": "showmehow",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--system-talk-name=com.endlessm.Metrics",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=host:ro",
+        "--filesystem=home",
+        "--talk-name=ca.desrt.dconf",
+        "--talk-name=com.endlessm.ShowmehowService",
+        "--talk-name=com.endlessm.CodingGameService",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--command=showmehow"
+    ],
+    "modules": [
+        {
+            "name": "coding-game-service",
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/share"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "T23089",
+                    "url": "git://github.com/endlessm/coding-game-service"
+                }
+            ]
+        },
+        {
+            "name": "showmehow-service",
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/share"
+            ],
+            "sources": [
+                {
+                    "branch": "T23090",
+                    "type": "git",
+                    "url": "git://github.com/endlessm/showmehow-service"
+                }
+            ]
+        },
+        {
+            "name": "showmehow",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --force --no-compile -O0 --single-version-externally-managed --root=/",
+                "mkdir -p $FLATPAK_DEST/share/glib-2.0/schemas",
+                "cp data/com.endlessm.showmehow.gschema.xml $FLATPAK_DEST/share/glib-2.0/schemas",
+                "glib-compile-schemas $FLATPAK_DEST/share/glib-2.0/schemas"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "@GIT_CLONE_BRANCH@",
+                    "path": "."
+                }
+            ]
+        }
+    ],
+    "runtime": "com.endlessm.apps.Platform",
+    "runtime-version": "4",
+    "sdk": "com.endlessm.apps.Sdk"
```

Comment:
Question: Not sure if we want to build against our own SDK or use the gnome or freedesktop one.